### ISSUE TITLE
Permissions?  What Am I Missing Here?

### ISSUE THREAD

**Issue by davisford** [ghenry22/Subsonic_Synology#38]
I have a brand new DS1618+ just setup last week.

![Screen Shot 2020-05-16 at 8 36 56 PM](https://user-images.githubusercontent.com/186677/82133079-49b9a780-97b5-11ea-9deb-65682a7e1f50.png)

![Screen Shot 2020-05-16 at 8 37 09 PM](https://user-images.githubusercontent.com/186677/82133080-4d4d2e80-97b5-11ea-8feb-afc59521880c.png)

![Screen Shot 2020-05-16 at 8 38 11 PM](https://user-images.githubusercontent.com/186677/82133081-52aa7900-97b5-11ea-99b9-7a4a82e2569e.png)

When I force a scan of the media folder....

![Screen Shot 2020-05-16 at 8 40 05 PM](https://user-images.githubusercontent.com/186677/82133088-71107480-97b5-11ea-960a-371271f3b617.png)

There's no data....

Logs =>

![Screen Shot 2020-05-16 at 8 40 43 PM](https://user-images.githubusercontent.com/186677/82133094-871e3500-97b5-11ea-9f64-be37b53db28a.png)

**Comment by davisford** [comment]
![Screen Shot 2020-05-17 at 12 16 17 PM](https://user-images.githubusercontent.com/186677/82153915-6d720180-9838-11ea-92c8-1a95f0db05c8.png)

The missing link was to tick the box `Apply to this folder, sub-folders and files` in their File Manager app.  After that Subsonic was able to index the media.

**Comment by ghenry22** [comment]
Thanks for the update, I'll add a note into the readme in case people hit this in future.

**Comment by mopey69** [comment]
I had the same problem and even all this didn’t work. synology permissions are so flakey. To resolve it for me I had to click on advanced permissions, check the subsonic group, select read only (even though I already did this under the normal permissions) the library folder then disappeared from /volume1. Then I unselected the above from advanced settings and hey presto it started to add the libraries.
